src/asserts/aba-account-number-assert.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { Validator, Violation } = require('validator.js');
+
+/**
+ * Account number regex.
+ */
+
+const accountNumberRegex = /^\d{8,17}$/;
+
+/**
+ * Export `AbaAccountNumberAssert`.
```

Comment:
Is this supposed to be specific to `ABA`?

Reply:
Yes.